### DISCUSSION TITLE
I lost the ability to modify the view columns after windows update

### DISCUSSION THREAD

**Discussion by hamzagad** [valinet/ExplorerPatcher#2450] [General]
Hi

I install a new fresh latest version of windows 11 Pro

Here is the build number:
Edition	Windows 11 Pro
Version	23H2
OS build	22631.2715
Experience	Windows Feature Experience Pack 1000.22677.1000.0

I was able to add a new column in the sort by, now I can't

![epm](https://github.com/valinet/ExplorerPatcher/assets/17599155/e7407c7f-b68e-4330-95d2-5053b958e5d1)

**Comment by hamzagad** [comment]
I found that if I changed the control interface to windows 10, I get my option back, but I lose the multiple tabs, i.e. each folder will open in a separate window

![epm2](https://github.com/valinet/ExplorerPatcher/assets/17599155/0264eabb-a0b3-4b60-bdd7-2a116c2d1373)

**Reply by Willie982** [reply]
Right click on any column header; Name, Date modified, Type, Size, etc.

![scrn](https://github.com/valinet/ExplorerPatcher/assets/151062840/f5304ac8-73fa-46fe-b403-bbb7a798d8f5)

**Comment by hamzagad** [comment]
Thank you :)
